Translations/translation_BG.json

```diff
@@ -289,6 +289,20 @@
 				"voltage"
 			],
 			"desc": "Minimum allowed voltage per cell <Volts> <3S — 3.0V - 3.7V, 4/5/6S — 2.4V - 3.7V>"
+		},
+		"PowerPulseWait": {
+			"text2": [
+				"Power pulse",
+				"wait time"
+			],
+			"desc": "Time to wait before triggering every keep-awake pulse (x 2500 ticks)"
```

Comment:
I feel it would be better to indicate 2.5 seconds rather than 2500 ticks?

Reply:
Agreed, it would be better to use s/ms and convert it to ticks using current frequency.